etc/nanomq_example.conf

```diff
@@ -702,6 +702,12 @@ bridges.mqtt.emqx1 {
 			# # This must present to enable the forwarding of bridging 
 			# # Value: String
 			local_topic = "topic1"
+			# # Retain is used to override the ratain flag in the msg is about
+			# # to forward to remote. Set 0 or 1 to override the ratain flag. 
+			# # Leave it as it is by not set it.
+			# # Value: 0 | 1
+			# # Default: not set
+			# # retain = 0
```

Comment:
Why not tell user retain=2 means not override

Reply:
nice try